vtr_flow/tasks/mixing_optimization/correctness_custom/mults_and_adders_simple/config/run_this_script_to_check_correctness.sh

```diff
@@ -0,0 +1,23 @@
+VTR_ROOT=`realpath $0`
```

Comment:
there has got to be a better way to hook into the vtr_flow. this is hacky

Reply:
Entire VTR relies on VTR_ROOT being set. Can use that?

Reply:
no I mean, can you add a pass to the vtr_flow itself to do odin simulation?

Reply:
either that, or just use our baked in verification tool, itll handle that for you

Reply:
This has something to do with odin simulator not being able to load blifs generated by abc, I do not know if that's still an issue. Would you suggest removing that file alltogether?

Reply:
i wouldnt test a before and after ABC, i care more about comparing to a predefined vector from a no modification flow to the modified version